kolibri/content/api.py

```diff
@@ -217,6 +219,18 @@ def filter_kind(self, queryset, name, value):
             return queryset.exclude(kind=content_kinds.TOPIC).order_by("lft")
         return queryset.filter(kind=value).order_by("lft")
 
+    def filter_in_exam(self, queryset, name, value):
+        """
+        Show only content associated with this exam
+
+        :param queryset: all content nodes
+        :param value: id of target exam
+        :return: content nodes for this exam
+        """
+        question_sources = Exam.objects.get(id=value).question_sources
```

Comment:
does this handle when the ID is not a valid exam id?

Reply:
What should be the correct error codes that are returned?
because there could be invalid exam id ex. `123`
and non-existent, but valid exam id

Reply:
I don't think there needs to return a 4xx error. I think if you asked for `in_lesson=bogusle55on`, it could just return an empty array.